qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/InstrumentWidget.h

```diff
@@ -43,6 +43,8 @@ class QDragEnterEvent;
 class QDropEvent;
 class QStackedLayout;
 class QSettings;
+class QVBoxLayout;
+class QHBoxLayout;
```

Comment:
Not sure you need this forward declaration of QHBoxLayout - but not worth worrying about if this is the only change